jest/private/jest_test.bzl

```diff
@@ -129,12 +139,28 @@ def _impl(ctx):
         for target in ctx.attr.data
     ])
 
-    return [
+    if ctx.configuration.coverage_enabled:
+        providers.append(
+            coverage_common.instrumented_files_info(
+                ctx,
+                source_attributes = ["data"],
+                extensions = [
+                    "mjs",
```

Comment:
add `mjx` and `cjx` and I think the typescript flavors as well incase jest is configured to transpile typescript for a user: `ts`, `tsx`, `cts`, ctx`, `mts`, `mtx`